packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -400,7 +401,6 @@ export enum FeatureFlagKey {
   IsSsoEnabled = 'IsSSOEnabled',
   IsStripeIntegrationEnabled = 'IsStripeIntegrationEnabled',
   IsUniqueIndexesEnabled = 'IsUniqueIndexesEnabled',
-  IsViewGroupsEnabled = 'IsViewGroupsEnabled',
```

Comment:
https://github.com/twentyhq/twenty/commit/b71246bc5dd54bdd48ab6f4bbc4c402f140f0d37 this was brought back under a feature flag